cvat-ui/src/components/header/settings-modal/styles.scss

```diff
@@ -19,7 +19,7 @@
     width: 100%;
     height: max-content;
     background: $background-color-1;
-    padding: 50px;
+    padding: 24px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5722655)